dogowner/migrations/0002_test_data.py

```diff
@@ -9,18 +10,52 @@ class Migration(migrations.Migration):
     def generate_data(apps, schema_editor):
         from dogowner.models import DogOwner
 
-        test_data = [
-            ('user1@address.com', 'testUser1', 'password123', 'dog name', 'first name', 'last name',
-             1234567890, 'dog race', 'https://www.google.com/user1', 4, 2, 'M'),
-            ('user2@address.com', 'testUser2', 'password123', 'dog name', 'first name', 'last name',
-             1234567890, 'dog race', 'https://www.google.com/user2', 3, 1, 'M'),
-            ('user3@address.com', 'testUser3', 'password123', 'dog name', 'first name', 'last name',
-             1234567890, 'dog race', 'https://www.google.com/user3', 2, 5, 'M'),
+        static_dog_owners = [
+            ('david00@address.com', 'David', 'password123', DOG_NAME_LIST[0], 'David', 'Chen',
+             4543837361, DOG_RACE_LIST[0], DOG_IMAGE_URL_LIST[0], 4, 10.3, 'M'),
```

Comment:
While nice to have lists of races, images, and names - having to look at 2 files to understand what I should expect is unreadable.

Reply:
NP I will make the changes .